doc/01-about.md

```diff
@@ -18,6 +18,7 @@ The module provides the following permissions:
 * `grafana/graphconfig`, Allow to configure graphs.
 * `grafana/graph`, Allow to view graphs in dashboards.
 * `grafana/showall`, Allow access to see all graphs of a host.
+* `grafana/showlink` Allow users to see a link to the Grafana instance.
```

Comment:
Slighly misleading. I would suggest `Display a link to the Grafana instance` or something similar.
Technically the permissions the permissions allows a users to see the link, but effectively is enables/disables the link button.
`allow` sounds to me like "now I can do something different/new" which is not really the case IMHO

Reply:
```suggestion
* `grafana/showlink` Display a link to the Grafana instance.
```